README.md

```diff
@@ -0,0 +1,60 @@
+# Mountsion CBS Secondary Website
+
+![alt text](https://github.com/MountSionCBSSecondary/SchoolWebsite/tree/master/assets/images/Mount Sion Crest.jpg)
+
+This project is a website that is being made by 4 students from Mountsion CBS Secondary along with the help of RedHat Waterford. As time goes on more students will resume the work that we havn't finished and will continue to keep the website up to date.
+
+## Getting Started
+
+If you wish to contribute to the project take some time to read our getting started guide here
```

Comment:
If you could change this to following `If you wish to contribute to the project take some time to read our [getting started tutorial](./docs/Tutorial.md).` It will provide a direct link to our doc, Sorry I was not clear in my last comment on this.

Reply:
Nah I should have just linked it, my bad.